lib/pbench/server/auth/keycloak_admin.py

```diff
@@ -50,7 +38,9 @@ def get_admin_keys(self, realm_name: str) -> List:
         :return: keys list
         """
         params_path = {"realm-name": realm_name if realm_name else self.user_realm}
-        keys = self._get(URL_ADMIN_KEYS.format(**params_path), data=None).json()
+        keys = self._get(
+            auth_urls.URL_ADMIN_KEYS.format(**params_path), data=None
```

Comment:
Since you're using this formatting pattern a lot, you might consider encapsulating it as I did in #2941, where the `get`/`post` callers provide an API key and parameter dict and the method does the formatting.